meson_init.sh

```diff
@@ -135,9 +135,10 @@ else
     echo "Unable to find meson toolchain file at $ARG_toolchain_file. Aborting." >&2
     exit 1
   else
-    cross_file=$(mktemp /tmp/toolchain.XXXXXX.txt)
+    cross_file="${OBJ_DIR}/toolchain-configured.txt"
```

Comment:
```suggestion
    cross_file="$OBJ_DIR/toolchain-configured.txt"
```

And you need to create the OBJ_DIR first by moving the lines below this block above it (run `./meson_init.sh -f` to check)